src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,109 @@
+package wolox.training.models;
+
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import wolox.training.exceptions.BookAlreadyOwnedException;
+
+/**
+ * The type User.
+ */
+@Entity
+@Table(name = "users")
+public class User {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.AUTO)
+  private long id;
+
+  @Column(nullable = false)
+  private String username;
+
+  @Column(nullable = false)
+  private String name;
+
+  @Column(nullable = false)
+  private LocalDate birthdate;
+
+  @ManyToMany(cascade = CascadeType.ALL)
+  @JoinTable(name = "user_book",
+      joinColumns = @JoinColumn(name = "book_id", referencedColumnName = "id"),
+      inverseJoinColumns = @JoinColumn(name = "user_id", referencedColumnName = "id")
+  )
```

Comment:
It is not necessary to use the @JoinTable annotation, JPA is magic and it generates the join table and its columns by itself. We only use @JoinTable when the strategy used by JPA to set the columns' names doesn't match the project naming conventions (it's not our case)